index.html

```diff
@@ -496,7 +517,7 @@
     document.getElementById('chat-notification-button').disabled = (permission.status == 'denied');
   }
 
-  navigator.permissions.query('notifications').then(function(result) {
+  navigator.permissions.query({name:'notifications'}).then(function(result) {
     updateNotificationButton(result);
```

Comment:
Given that `result` is basically just `status` + an event handler, maybe just pass `result.status` here and below.